docs/examples/config/lp_custom.rst

```diff
@@ -111,22 +115,22 @@ An example YAML configuration file for the OGBN_Arxiv dataset (link prediction m
 
 Let's create the same YAML configuration file for the OGBN_Arxiv dataset from scratch. We follow the structure of the configuration file and create each of the four sections one by one. In a YAML file, indentation is used to denote nesting and all parameters are in the format of key-value pairs. 
 
-#. First, we define the **model**. We begin by setting all required parameters. This includes ``learning_task``, ``encoder``, ``decoder``, and ``loss``. Since we are training a link prediction model, set the ``learning_task`` to ``LINK_PREDICTION``. Since we are training a model with DistMult, set the ``type`` of ``decoder`` to ``DISTMULT``. We set the ``encoder`` to be an ``EMBEDDING`` table with 50-dimensional embeddings. The rest of the configurations can be fine-tuned by the user.
+#. First, we define the **model**. We begin by setting all required parameters. This includes ``learning_task``, ``encoder``, ``decoder``, and ``loss``. The rest of the configurations can be fine-tuned by the user.
 
     .. code-block:: yaml
     
         model:
-          learning_task: LINK_PREDICTION
+          learning_task: LINK_PREDICTION # set to "LINK_PREDICTION" since we train a link prediction model
```

Comment:
Make a not here or in the above text that these string values are case insensitive but we use capital letters for convention.

Also the comments are a bit redundant 

Can just say:
"set the learning task to link prediction"